capstone/src/main/webapp/index.html

```diff
@@ -3,13 +3,16 @@
   <head>
     <title>UniViz Main Page</title>
     <meta charset="utf-8">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js" type="text/javascript"></script>
   </head>
   <body>
     <h2>Welcome to UniViz!</h2>
     <form autocomplete="off">
       <label for="search">Search for colleges:</label><br>
-      <input type="text" id="search" name="search"><br>
+      <input type="text" id="search" name="search" placeholder="College Name"><br>
       <input type="submit" value="See your colleges!">
     </form>
   </body>
+  <script src="js/autocomplete.js"></script>
+  <script src="js/autocompleteTest.js"></script>
```

Comment:
```suggestion
```
Unless there's some function in `autocompleteTest.js` that you need inside this file.